### DISCUSSION TITLE
Saving User Registration

### DISCUSSION THREAD

**Discussion by alexmayerlab** [go-acme/lego#2438] [Q&A]
https://go-acme.github.io/lego/usage/library/

In this example, registration occurs every time, and if I add 20 domains every hour, then limits are possible,
how to get around this? I think you need to save your account and use it when registering new domains?

**Comment by ThinkChaos** [comment]
I know this is very old, but you can look at how lego does it: https://github.com/go-acme/lego/blob/v4.20.0/cmd/accounts_storage.go
